client/src/app/routes/stardust/Addr.tsx

```diff
@@ -738,7 +738,7 @@ class Addr extends AsyncComponent<RouteComponentProps<AddrRouteProps>, AddrState
                 this._bechHrp,
                 this._tangleCacheService);
         const inputsRelated = inputs.filter(input => input.transactionAddress.hex === this.state.address);
-        const outputsRelated = outputs.filter(output => output.address.hex === this.state.address);
+        const outputsRelated = outputs.filter(output => output);
```

Comment:
This looks weird now. If this is filter predicate, then we don't need to filter i guess.

Reply:
Changed filter to map js function